section2/reflection.md

```diff
@@ -1,15 +1,25 @@
+  * I did not notice that the directions were left out. After last lesson and having spend a good amount of time becoming comfortable with completing similar tasks I did not have to think much about how am I going to run this. I think part of this comes from the mistakes I made on the last part of the project. I thought I had to push and commit each change I made before I ran node. This led to me having many repetitions of each of those commands.
+  * If I do have to say one thing about the experience is that it felt like more of a challenge, having to recall the information from memory. I find to this to be a helpful thing during this exercise. It challenged me to make sure I was able to recall the information and if not find where the information is located.
 
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
-
+  * A conditional statement allow the program to run different blocks of code depending on defined boolean variables being true or false.
+  * What does the question mean "daily life conditional"? Work starts at 9, I need 30 minutes to get ready. If I wake up with less than 30 minutes until 9 I will be late. Thats a "conditional" outside of programing if that is what is being asked.
+  * On Youtube if you have an account you will have a "Liked" video section that saves all your liked videos to that area. A code maybe able to determine if a video is liked by user than moved to like folder. If video is not liked by user video is not added to folder.
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+  * You can use AND and OR operators. AND is && OR is ||. If the logical operators are used you can have them check multiple variables, This AND that have to be true or this OR that has to be true. The program will only return true when using && if both of the operants are true, while || only returns false when both of its operants are false.
 
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
-
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+  * Would making a flow chart be a tool you can use? Looking over the data yourself you maybe able to determine if the conditions are met. You can use the console within chrome to test the code using Javascript.
+7. What questions do you still have about `if` statements and/or functions?
+  * After reviewing the assigned book and class material I am wondering what tools are available to check the code before running it. Is there data that a `if` statement can not check?
```

Comment:
As far as tools - JavaScript is language that is interpreted at tun-time, so you need to run it to check it. You can use tools like repl.it, node in the Terminal or the browser dev tools!

RE Is there data an if cannot check - an if statement evaluates a condition - it needs to be provided with something that can be evaluated down to true or false. JavaScript will try to "help" you if you don't give it something that does that, by looking for truthy or falsey values - you'll learn about in Mod 1 a bit and probably become well versed by Mod 2, but feel free to look into earlier :)